java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/SampleDelegateCommandHandler.java

```diff
@@ -50,6 +50,43 @@ private void waitForJavaSourceDownloads() {
     // to filter.
     private static SearchPattern mapLocationToSearchPatternLocation(int location, String query) throws Exception {
         //TODO: #21 Normalize queries and/or verify for each location.
+
+        if (query.contains("(") || query.contains(")")) {
+            // We know that this is a list of things to loook for, broken by a | command. We should get this intra string and create an OR search pattern for each one. 
+            // ex ((FileWriter|FileReader|PrintStream|File|PrintWriter|RandomAccessFile))
+            var startListIndex = query.indexOf("(");
+            var endListIndex = query.indexOf(")");
+            logInfo("Start index: " + startListIndex + " end index: " + endListIndex);
+            var startQuery = query.substring(0, startListIndex);
+            var endQuery = query.substring(endListIndex+1, query.length());
```

Comment:
What are `startQuery` and `endQuery` for?

Reply:
https://github.com/konveyor/java-analyzer-bundle/pull/33#discussion_r1121939597 I think answers this?